src/vs/base/test/node/terminalEncoding.test.ts

```diff
@@ -0,0 +1,23 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the MIT License. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+
+import * as assert from 'assert';
+import * as terminalEncoding from 'vs/base/node/terminalEncoding';
+import * as encoding from 'vs/base/common/encoding';
+
+suite('Terminal encoding', () => {
```

Comment:
Because I move `encoding.test.ts` to live in `common` I've had to extract terminal encoding tests to keep them in `node`.

Reply:
Again, I think these tests have to remain in `node`, at least the ones that need `fs`